### PR TITLE
POC Dockerized Discovery image in RACKHD

### PR DESCRIPTION
**Pull request by keedya**
@RackHD/corecommitters 

depends on : https://github.com/RackHD/on-taskgraph/pull/135
depends on : https://github.com/RackHD/on-http/pull/350

### CODE REVIEW

File: lib/task-data/tasks/bootstrap-rancher.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        initrdFile: 'initrd',
+        kernelUri: '{{ api.server }}/common/{{ options.kernelFile }}',
+        initrdUri: '{{ api.server }}/common/{{ options.initrdFile }}',
+        profile: 'linux.ipxe',
+        comport: 'ttyS0'
+    },
+    properties: {
+        os: {
+            linux: {
+                distribution: 'ubuntu',
```
Comment (benbp) [inline lib/task-data/tasks/bootstrap-rancher.js]: Properties should be changed.

File: lib/task-data/base-tasks/linux-bootstrapper.js [hunk 2]
```diff
@@ -11,8 +11,6 @@ module.exports = {
         'initrdFile',
         'kernelUri',
         'initrdUri',
-        'basefs',
-        'overlayfs',
```
Comment (benbp) [inline lib/task-data/base-tasks/linux-bootstrapper.js]: FYI @yyscamper, I assume these are in/going in the task schemas? Will want to make them optional values.

**Comment by brianparry** [comment]
test this please

**Comment by keedya** [comment]
closing POC and open a new PR with fianl changes
https://github.com/RackHD/on-tasks/pull/293

**Pull request closed without merging**